dlrover/examples/torch_mnist_master_backend_job.yaml

```diff
@@ -21,11 +21,20 @@ spec:
                 - /bin/bash
                 - -c
                 - "python -m dlrover.python.elastic_agent.torch.prepare \
-                && torchrun --nnodes=1:$WORKER_NUM --nproc_per_node=1
-                --max_restarts=3 --rdzv_backend=dlrover-master \
-                model_zoo/pytorch/mnist_cnn.py \
-                --training_data /data/mnist_png/training/elastic_ds.txt \
-                --validation_data /data/mnist_png/testing/elastic_ds.txt"
+                  && torchrun --nnodes=1:$WORKER_NUM --nproc_per_node=1
+                  --max_restarts=3 --rdzv_backend=dlrover-master \
+                  model_zoo/pytorch/mnist_cnn.py \
+                  --training_data /data/mnist_png/training/elastic_ds.txt \
+                  --validation_data /data/mnist_png/testing/elastic_ds.txt"
+              resources:
+                limits:
+                  cpu: "1"
+                  memory: 2Gi
+                  nvidia.com/gpu: 1
```

Comment:
Remove GPU because users may run the example without GPU.

Reply:
already removed now.